reqs/requirements-dev-bigquery.txt

```diff
@@ -1,5 +1,9 @@
 gcsfs>=0.5.1
 google-cloud-bigquery>=3.3.6
+# needed to prevent excessive pip backtracking on pythonn 3.11
+google-cloud-bigquery-storage>=2.20.0;python_version >= '3.11'
```

Comment:
Without this, I still get way too much backtracking (as in +30 minutes)
Even with this, the build step is incredibly long but it least it doesn't hit the CI build timeout limit.
```
[+] Building 1039.9s (15/15) FINISHED  
```

Reply:
hmm what's the reason for this - is GCS just not well equipped for 3.11 or lacks official support?